src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -189,15 +189,19 @@ def __init__(self):
 
     def _verify_job_id(self, req: JobRequest) -> None:
         if not req.job_id:
-            self.send_error_message("job_does_not_exist", req)
+            self.send_error_message(
```

Comment:
there's no way to tell the difference between a request with no input and one with invalid input, so this fixes that issue.